charts/civil-service/values.yaml

```diff
@@ -43,6 +43,7 @@ java:
     DOCMOSIS_TORNADO_URL: https://docmosis.{{ .Values.global.environment }}.platform.hmcts.net
     DOCUMENT_MANAGEMENT_URL: http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    CLAIM_STORE_URL: http://cmc-claim-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
You might need to add same env variable  in https://github.com/hmcts/civil-citizen-ui/blob/master/charts/civil-citizen-ui/values.preview.template.yaml#L85.

Reply:
yes, that would be done in the corresponding story https://tools.hmcts.net/jira/browse/CIV-2747 in the citizen ui branch